webrtc-stats.html

```diff
@@ -995,6 +995,8 @@ <h3>
              DOMString            decoderImplementation;
              DOMString            playoutId;
              boolean              powerEfficientDecoder;
+             double               totalAssemblyTime;
```

Comment:
Should this be totalFrameAssembly Time?
Possibly this reads better if it follows the framesAssembledFromMultiplePackets definition.